src/site/content/en/blog/use-thumbor/index.md

```diff
@@ -0,0 +1,217 @@
+
+
+## Prequisites
+
+This post assumes that you understand how image CDNs can improve your load performance. If not, check out [Use image CDNs to optimize images](https://web.dev/image-cdns). It also assumes that you've built basic websites before.
+
+{% Aside %}
+
+If you would like to install Thumbor on your own server and then follow along with this post, check out [How to install the Thumbor image CDN](https://web.dev/install-thumbor). Whenever you see `http://34.67.235.246:8888` in this post you'll need to replace that origin with your Thumbor instance's origin.
+
+{% endAside %}
+
+
+## Thumbor URL Format
+
+As mentioned in [Use Image CDNs to Optimize Images](https://web.dev/image-cdns), each image CDN uses a slightly different URL format for images. Figure 1 represents Thumbor's format.
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor0.jpg" alt="A Thumbor URL has the following components: origin, security key, size, filters and image." class="w-screenshot">
+</figure>
+
+_Figure 1. Thumbor's URL format_
+
+
+### Origin
+
+Like all [origins](https://html.spec.whatwg.org/multipage/origin.html#concept-origin), the origin of a Thumbor URL is composed of three parts: a [scheme](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#Definition) (which is almost always `http` or `https`), a host, and a port. In this example, the host is identified using an IP address, but if you're using a DNS server it might look like `thumbor-server.my-site.com`. By default, Thumbor uses port `8888` to serve images.
+
+
+### Security Key
+
+The `unsafe` part of the URL indicates that you're using Thumbor without a [security key](https://github.com/thumbor/thumbor/wiki/security). A security key prevents a user from making unauthorized changes to your image URLs. By changing the image URL, a user could use your server (and your hosting bill) to resize their images, or, more maliciously, to overload your server. This guide won't cover setting up Thumbor's security key feature.
+
+
+### Size
+
+This part of the URL specifies the desired size of the output image. This can be omitted if you don't want to change the size of the image. Thumbor will use different approaches like cropping or scaling to achieve the desired size depending on the other URL parameters. The next section of this post explains how to resize images in more detail.
+
+Try it now:
+
+1. Click the URL below to view the image served at its original size. The image opens in a new tab.
```

Comment:
```suggestion
1. Click the following URL to view the image served at its original size in a new tab: <a href="http://34.67.235.246:8888/unsafe/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/https://web.dev/backdrop-filter/hero.jpg</a>
```

Now that I see the rendered article, I realize it's a little awkward to have the URL below the instructions. Please use the format above for all the instructions. So each instruction should end with a colon, and then we list the URL to be clicked